### DISCUSSION TITLE
How to build a separate protoc-c binary file without relying on any other libraries?

### DISCUSSION THREAD

**Discussion by xuning0** [protobuf-c/protobuf-c#609] [Q&A] [answered]
I have tried to build according to documentation, but I just got a `protoc-gen-c`. It looks to depend on `protobuf` and `libprotoXXX.dylib` on my machine.

How can I  get a separate protoc-c binary, which can be copied to any other machines and exec `protoc-c --c_out=xxxxxxx` directly?

**Comment by xuning0** [comment]
It sounds weird, but I use our company's common CI environment, so I may only add the `protoc-c` binary file to my project.

**Comment by lipnitsk** [comment, marked as answer]
No, protoc-c requires libprotobuf.
